src/Deserializers/StatementDeserializer.php

```diff
@@ -81,8 +81,20 @@ private function hasCorrectType( $serialization ) {
 	 * @return Statement
 	 */
 	public function deserialize( $serialization ) {
-		$this->assertCanDeserialize( $serialization );
-		$this->requireAttribute( $serialization, 'mainsnak' );
+		// Note: The following functions have been inlined on purpose, see T157013.
```

Comment:
If you link the issue than this is not cryptic to anyone not familiar with which issue tracking tool WMF uses